vllm/model_executor/models/deepseek.py

```diff
@@ -33,7 +33,7 @@
                               get_tensor_model_parallel_world_size,
```

Comment:
Could you add test cases for the change?

Reply:
How to test this change? model e2e test?

Reply:
Yes, you could add a model test case, similar to the pattern followed here: https://github.com/vllm-project/vllm/tree/main/tests/models